packages/app-project/src/helpers/getDefaultPageProps/getDefaultPageProps.js

```diff
@@ -7,12 +7,13 @@ export default async function getDefaultPageProps({ params, query, req, res }) {
   // cookie is in the next.js context req object
   const mode = getCookie(req, 'mode') || null
   const dismissedAnnouncementBanner = getCookie(req, 'dismissedAnnouncementBanner') || null
-  const store = initStore({
+  const snapshot = {
     ui: {
       dismissedAnnouncementBanner,
       mode
     }
-  })
+  }
+  const store = initStore(true, snapshot)
```

Comment:
I'd prefer `true` be set to a variable to clearly show what this parameter is for. Perhaps:
```suggestion
  const isServer = true
  const store = initStore(isServer, snapshot)
```